src/state.c

```diff
@@ -2824,3 +2844,70 @@ getNetRxTxBucket(net_info *net)
 
     return bucket;
 }
+
+// Create a security event for a file
+void
+fileSecurity(const char* path, const char* reason, bool close, uint64_t write_bytes)
```

Comment:
To be consistent vars should be type *name as opposed to type* name. Minor, yet annoying.

Reply:
Fixed